scripts/circle-ci/run-circle-tests.sh

```diff
@@ -22,6 +22,8 @@ CONTAINER_5=(':atlasdb-ete-tests:longTest' ':lock-impl:check' ':atlasdb-dbkvs-te
 
 CONTAINER_6=(':atlasdb-ete-test-utils:check' ':atlasdb-cassandra:check' ':atlasdb-api:check' ':atlasdb-jepsen-tests:check' ':atlasdb-cli:check')
 
+CONTAINER_7=('checkstyle{Main,Test} -PenableErrorProne=true')
```

Comment:
This should be `compileJava/compileTestJava`, checkStyle is run by the build task in other containers as well.